features/src/main/feature/camel-features.xml

```diff
@@ -1884,12 +1884,12 @@
 <!--        <bundle dependency='true'>mvn:commons-io/commons-io/${commons-io-version}</bundle>-->
 <!--        <bundle>mvn:org.apache.camel/camel-salesforce/${project.version}</bundle>-->
 <!--    </feature>-->
-<!--    <feature name='camel-saxon' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-core</feature>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xmlresolver/${xmlresolver-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.saxon/${saxon-bundle-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-saxon/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-saxon' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xmlresolver/${xmlresolver-bundle-version}</bundle>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.saxon/${saxon-bundle-version}</bundle>
```

Comment:
Those 2 properties are not defined in the project so it cannot build

Reply:
properties are added to pom.xml